docs/dashboards/custom-chart.md

```diff
@@ -0,0 +1,164 @@
+# Custom Chart
+
+The Custom Chartm Block is an advanced Block that allows you to create custom data visualization by utilizing [Vega-Lite](https://vega.github.io/vega-lite/). If you would like a specific visualization that's not already a Losant Dashboard Block, the Custom Chart Block gives you the ultimate support for a wide array of visualization types.
+
+![Data Table Scatter Plot](../images/workflows/custom-chart-data-table-scatter-plot.png "Data Table Scatter Plot")
+
+## Configuration
+
+Configuring the Custom Chart Block is broken up into three sections ...
+
+### Choosing an Application
+
+First, choose an application that owns the devices and/or the data tables you would like to query.
+
+![Select Application](/images/workflows/custom-chart-select-application.png "Select Application")
+
+### Datasets
+
+![Datasets](/images/workflows/custom-chart-datasets.png "Datasets")
+
+#### Time Series Datasets
+
+Time series datasets provide the value of a single attribute from a selected device over a duration of time at a selected resolution. The values returned from the query will be returned as an array of objects in the form of `{ time: <time>, value: <value> }`. The value returned will reflect the selected attribute's value specified at a specific time.
+
+![Time Series Query](/images/workflows/custom-chart-time-series-query.png "Time Series Query")
```

Comment:
We talked about this IRL, but we should enumerate all the options for each query. 

I really like how it's done in the time series graph ( you can steal a lot from there ): 

/dashboards/time-series-graph/

Reply:
For the Data Table Query do you think it would be more helpful to refer to https://docs.losant.com/workflows/data/table-get-rows/#query-fields instead of listing out the parameters?

Reply:
Also, if I include a statement at the beginning of the query section stating the purpose of the Query Name parameter do you think I should still include it in the list of parameters for each query type?